MeshLib/MeshGenerators/LayeredVolume.cpp

```diff
@@ -58,6 +59,7 @@ bool LayeredVolume::createRasterLayers(const MeshLib::Mesh &mesh,
 	this->_minimum_thickness = minimum_thickness;
 	_nodes = MeshLib::copyNodeVector(bottom->getNodes());
 	_elements = MeshLib::copyElementVector(bottom->getElements(), _nodes);
+	_materials.insert(_materials.begin(), _elements.size(), 0); 
```

Comment:
I think that `insert` is the wrong function here.
Given a vector `a = {0,1,2,3}`; After `a.insert(a.begin(), 3, -1)` a is equal to {-1,-1,-1,0,1,2,3}. There are other functions to _fill_ a vector or to _resize_ a vector.